sdk/src/store.rs

```diff
@@ -2419,6 +2443,67 @@ pub mod tests {
             .expect_err("Should not verify");
     }
 
+    #[test]
+    #[cfg(feature = "file_io")]
+    fn test_unknown_asset_type_generation() {
+        // test adding to actual image
+        let ap = fixture_path("unsupported_type.txt");
+        let temp_dir = tempdir().expect("temp dir");
+        let op = temp_dir_path(&temp_dir, "unsupported_type.txt");
+
+        // Create claims store.
+        let mut store = Store::new();
+
+        // Create a new claim.
+        let claim1 = create_test_claim().unwrap();
+
+        // Create a new claim.
+        let mut claim2 = Claim::new("Photoshop", Some("Adobe"));
+        create_editing_claim(&mut claim2).unwrap();
+
+        // Create a 3rd party claim
+        let mut claim_capture = Claim::new("capture", Some("claim_capture"));
+        create_capture_claim(&mut claim_capture).unwrap();
+
+        // Do we generate JUMBF?
+        let signer = temp_signer();
+
+        // Move the claim to claims list. Note this is not real, the claims would have to be signed in between commmits
+        store.commit_claim(claim1).unwrap();
+        store.save_to_asset(&ap, &signer, &op).unwrap();
```

Comment:
I'd like to see this test verify that it actually went to an external file. I see the `save_to_asset` / `load_from_asset` pairing, but this test could just as easily pass for a file format where the manifest were embedded, right?

Reply:
How do you embed at manifest in a txt file?

Reply:
Yes, Eric's point is valid but not for this test.  I intentionally chose a format that will never contain a manifest.